lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -703,25 +704,38 @@ class KymoTrackGroup:
 
     def __init__(self, kymo_tracks):
         self._src = kymo_tracks
-        if self:
-            if len(set(kymo_tracks)) != len(kymo_tracks):
-                raise ValueError(
-                    "Some tracks appear multiple times. The provided tracks must be unique."
-                )
+        self._kymos = self._validate_compatible_sources()
 
-            self._validate_single_source(kymo_tracks)
+    def _validate_compatible_sources(self, *additional_tracks):
+        """Check that source kymos for all tracks (including in self) are compatible.
 
-    def _validate_single_source(self, kymo_tracks):
-        kymos = set([track._kymo._id for track in kymo_tracks])
-        channels = set([track._channel for track in kymo_tracks])
+        Parameters
+        ----------
+        *additional_tracks : KymoTrackGroup
```

Comment:
Why are you passing this as `*args`? Do we need support for multiple here? Wouldn't a default parameter with an empty tuple suffice?

Reply:
I think just my knee jerk reaction to not using iterators as default kwargs, but since `tuple` is immutable works just as well